### PR TITLE
Fix 9215 central heat pump schedule

### LINKED ISSUE #9215: CentralHeatPumpSystem Control Schedule default value and warnings when it does not find a match to the schedule
Issue overview
--------------
In resolving a helpdesk ticke (#16072) about why an idf shows zero operation results for CentralHeatPumpSystem while the simulation finished running fine, it was found that if there is an unmatched schedule from the input, the program would not give a warning message and even worse, use an default zero value to replace it. Since this control value is something like an availability schedule, it would be the best to put a default value of “1” instead. Also, adequate warning should be given to this schedule unmatch scenario in the err file. 

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version): All
 - Version of EnergyPlus (if using an intermediate build, include SHA): v9.4 to v9.6
 - Helpdesk ticket number: 16072 

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (list location of defect file [here](https://github.com/NREL/EnergyPlusDevSupport/tree/master/DefectFiles/9000s/9215))
 - [x] Ticket added to Pivotal for defect (development team task)
 - [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by jcyuan2020**
Pull request overview
---------------------
 - Fixes #9215 
 - Deal with the scenario when the central heat pump system control schedule could not found a match. In the original code this scenario will yield a "zero" default which will shut off the equipment, and without any warning messages. With the change in the current PR, a few warning messages will be given, and also an "AlwaysOn" treatment for this scenario is used instead of the "zero" default.  

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by Myoldmopar** [comment]
Is this still draft or ready to go?

**Comment by jcyuan2020** [comment]
Set to ready for view.

File: tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc]: This `compPtr` is unused and throwing a warning from the compiler.  I'm guessing you probably don't need to call the PlantComponent wrapper function anyway though, and instead just call the `GetWrapperInput` function instead (which the factory probably does anyway...)

File: tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc:363]: Reasonable assertion for the unit test :+1:

File: src/EnergyPlus/PlantCentralGSHP.cc
Comment (Myoldmopar) [inline src/EnergyPlus/PlantCentralGSHP.cc:717]: Yeah, this is fine with me.

**Review by Myoldmopar — CHANGES_REQUESTED** [review]
The change looks good, and the unit test seems sufficient, but it does need a small tweak.  At least the removal of the unused variable, and more preferable just not calling the factory at all, per my comment.  Please address that either way and push it up and this should be ready to go.  Also, is there a defect file for this?

File: tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc
Comment (jcyuan2020) [reply tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc]: Defect file added.

Comment (jcyuan2020) [reply tst/EnergyPlus/unit/PlantCentralGSHP.unit.cc]: The change is also done now. It calling the "GetWrapperInput()' function directly, with some minor changes in the idf snippet, e.g. added a schedule.

File: src/EnergyPlus/PlantCentralGSHP.cc
Comment (jcyuan2020) [reply src/EnergyPlus/PlantCentralGSHP.cc:717]: Thanks for the review! The changes have been pushed, waiting for the ci results.

**Comment by Myoldmopar** [comment]
Windows build failed but unrelated to this, it's fine.  Develop has moved a good amount since the last run so I'd want to do at least a local test/regression run before merging, but I think this should still be ready to go in, right?

**Comment by jcyuan2020** [comment]
@Myoldmopar Yes, it should be good to go. Would you like me to push a commit that merged with most recent develop or you want to do that locally on your PC?

**Comment by Myoldmopar** [comment]
I pulled develop into the branch locally and am building/testing.  If it is all good it'll merge here in a bit.  Thanks.

**Comment by Myoldmopar** [comment]
All tests pass locally with develop pulled in, including regressions which are fully clean.  Merging this, thanks @jcyuan2020

**Pull request merged**
